Dockerfile.dev

```diff
@@ -0,0 +1,41 @@
+FROM ubuntu:18.04
+
+ENV DEBIAN_FRONTEND noninteractive
+ENV NODE_VERSION "v12.13.0"
```

Comment:
**question (non-blocking):** Should uploader use node version 16? 🤔

Reply:
Yes, we are using Node v16 in `Dockerfile`. I _think_ `Dockerfile.dev` is where @krystophv moved the old Dockerfile we used to run Uploader on non-Ubuntu versions of Linux: https://github.com/tidepool-org/uploader/?tab=readme-ov-file#docker-for-linux

AFAIK that is broken since we started using Keycloak. With UiW Linux users won't have to build their own Uploader anymore - they can just use it in the browser.

I